flink-ml-lib/src/test/java/org/apache/flink/ml/classification/KnnTest.java

```diff
@@ -179,6 +183,42 @@ public void testFitAndPredict() throws Exception {
         verifyPredictionResult(output, knn.getLabelCol(), knn.getPredictionCol());
     }
 
+    @Test
+    public void testDataTypes() throws Exception {
```

Comment:
nits: would it be more intuitive to rename this test as `testInputTypeConversion()`?

Same for `LinearSVCTest`.